### ISSUE TITLE
No option for background transparency

### ISSUE THREAD

**Issue by LucySparkles** [LiveSplit/LiveSplit#1089]
There is an option for opacity but this also makes the timers and text transparent. I want the timers and text visible while the background itself is transparent. This seems like an obvious feature to have as not every runner has the luxury of two screens and can use OBS and color key to solve this problem.

**Comment by wooferzfg** [comment]
This isn't really possible without a resource intensive workaround, and I don't think OBS could capture it like that anyway.

**Comment by LucySparkles** [comment]
That really sucks because I love the autosplitting feature. It seems like an oversight to me because I can't even do something like put a transparent image as the background. I guess I'll have to live with it.

**Comment by CryZe** [comment]
In the future you can use LiveSplit One as a transparent browser source. Maybe we could add the browser source to the original LiveSplit as well at some point. So this should somewhat be solved in the future. Making the actual window transparent is something that we tried, but it never really worked out without taking way too many resources, so I don't see it happening that way.

**Comment by LucySparkles** [comment]
Would using LiveSplitOne's transparent browser source be viable on my 1 screen setup though? Would I be able to have the timer open over my game? Or is it again an OBS thing that you need 2 screens for? I feel like Llanfair has a background transparency feature in it (unless I'm insane and I'm remembering wrong) which is why it seemed silly that there was no way to do this with LiveSplit.

**Comment by CryZe** [comment]
I think with LiveSplit One we can also have proper transparent backgrounds. I experimented with this at some point and it worked fairly well. So that could be the solution for your problem.

**Comment by LucySparkles** [comment]
So what you're saying is I should download LiveSplit One and look into it a bit?

**Comment by CryZe** [comment]
No, we don't even have a Desktop Build yet (the old experimental one got scrapped), so this is not anywhere ready to be used.

**Comment by LucySparkles** [comment]
Oh alright. I'll definitely be waiting for that to launch as it is a tad annoying trying to run RE7 and not being bale to see the corner of my screen but I'll make do with what's available at the moment.
